### PR TITLE
flooding sparks and dots

### PR DESCRIPTION
**Pull request by jordansread**
colors need some work, but the functionality is there. 

image

### CODE REVIEW

**Comment by jordansread** [comment]
for #116

File: scripts/process/process.discharge_sparks.R [hunk 1]
```diff
@@ -31,7 +27,61 @@ process.discharge_sparks <- function(viz = as.viz('discharge-sparks')){
...
+  }, height=0.4, width=2)
+  y.out <- xml2::xml_attr(xml2::xml_find_first(x, '//*[local-name()="line"]'),'y1')
+  if (is.na(y.out)){
+    return("0")
+  } else {
+    return(y.out)
+  }
+}
+
+# Should rename at some point. Not always going to be discharge.
```
Comment (dblodgett-usgs) [inline scripts/process/process.discharge_sparks.R:57]: Could remove this comment now!

**Comment by dblodgett-usgs** [comment]
This looks great! 👍

**Comment by jordansread** [comment]
We should leave this open if we want to use it. There is a need to redo the mouseovers on the sites

**Comment by jordansread** [comment]
actually, merging so we can see on dev. Got 👍 from blodg

**Pull request merged**